test/utils/Utils.t.sol

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.23;
+
+import { Test, console2 } from "forge-std/Test.sol";
+import { Utils } from "script/contracts/Utils.sol";
+
+contract UtilsTest is Test {
+    Utils utils;
+
+    function setUp() public {
+        vm.setEnv("NETWORK", "testnet");
```

Comment:
is it possible to test both mainnet and testnet?

Reply:
I tried it from within the same `UtilsTest` contract, modifying environment from a test_ function but guessing that they are called in parallel, causing the environment to change during execution that is resulting in unexpected environment configuration causing random failures. Environment is shared even between test contracts for the way I'm executing tests, tried with subclass to UtilsTest, resulting in same random failures on execution caused by shared environment.

```solidity
import { UtilsTest } from "./Utils.t.sol";
import { Utils } from "script/contracts/Utils.sol";

contract UtilsTestDevnet is UtilsTest {
    function setUp() public override {
        network = "devnet";
        vm.setEnv("NETWORK", network);
        utils = new Utils();
    }
}

```

Reply:
Perhaps it's not needed to test both networks?
Unit tests for `Utils.sol` are meant to just test if functions inside it are working correctly.
So probably this can be tested independently of which data set we use?